packages/cozy-harvest-lib/src/components/OAuthForm.jsx

```diff
@@ -70,8 +70,11 @@ export class OAuthForm extends PureComponent {
   }
 
   render() {
-    const { konnector, t, flowState, reconnect, account } = this.props
+    const { konnector, t, flowState, reconnect, account, intentsApi } = this.props
     const { showOAuthWindow, needExtraParams, extraParams } = this.state
+    const { konnector, t, flowState, intentsApi } = this.props
+    const { initialValues, showOAuthWindow, needExtraParams, extraParams } =
+      this.state
```

Comment:
on peut fusionner ici il y a 2 destruc de this.state et 2 destruct de this.props